tutorials/plot_measures.py

```diff
@@ -11,40 +11,124 @@
 # Applying measures & conversions to spiking data
 # -----------------------------------------------
 #
-# Words, words, words.
+# Sections
+# ~~~~~~~~
+#
+# This tutorial contains the following sections:
+#
+# 1. Compute measures of spiking activity
+# 2. Convert spiking data
 #
 
 ###################################################################################################
 
 # sphinx_gallery_thumbnail_number = 1
 
-import spiketools
+# import auxiliary libraries
+import numpy as np
+import matplotlib.pyplot as plt
+
+# import all functions from spiketools.measures
+from spiketools.measures.measures import *
+from spiketools.measures.conversions import *
 
 ###################################################################################################
 
-# Settings
-...
+# Compute measures of spiking sctivity
+# ~~~~~~~~
+
+# First, we estimate spike rate from a vector of spike times, in seconds.
+# 
 
 ###################################################################################################
-# Subtitle
-# ~~~~~~~~
-#
-# Words, words, words.
+
+# set and visualize a series of spike times
+spikes = [0.1, 0.3, 0.9, 1.9, 2.0, 2.1, 2.2, 3.4, 3.5, 3.7, 4.6, 4.8, 4.9, 7]
+plt.eventplot(spikes)
+
+###################################################################################################
+
+# compute its spike rate
+print('The spike rate is', compute_spike_rate(spikes))
+
+###################################################################################################
+
+# Then, we can compute the inter-spike intervals of that vector of spike times.
 #
 
 ###################################################################################################
 
-# Do some code stuff
-print('Hello World!')
+isis = compute_isis(spikes)
+print('The inter-spike intervals are', isis)
```

Comment:
ISIs are a distribution of multiple values, and in general we wouldn't necessarily want to print them all out. 

More typically, we might want to plot the ISI distribution, which you can do with `spiketools.plts.spikes.plot_isis`